ArcadiaTeamsBot/Infrastructure/Bot.cs

```diff
@@ -37,7 +39,7 @@ protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersA
                 if (member.Id != turnContext.Activity.Recipient.Id)
                 {
                     await turnContext.SendActivityAsync(
-                        $"Welcome to ArcadiaBot, {member.Name}.",
+                        $"Welcome to ArcadiaBot, {member.Name}. {WelcomeText}",

```

Comment:
It is better to have one constant string with a placeholder like `Welcome ...,  {username} ...` and replace `{username}` with correct value.

Reply:
Fixed